rocAL/rocAL/source/meta_data/coco_meta_data_reader.cpp

```diff
@@ -200,8 +252,10 @@ void COCOMetaDataReader::read_all(const std::string &path)
             parser.EnterArray();
             while (parser.NextArrayValue())
             {
-                int id = 1, label = 0;
+                int id = 1, label = 0, iscrowd = 0;
                 std::array<float, 4> bbox = {};
```

Comment:
Use double here since BoundingBoxCord and all other functions expect double dtype